site/style_about.css

```diff
@@ -0,0 +1,18 @@
+
+body {
+	border:solid 5px;
+	padding-left: 1.5cm;
+	padding-right: 1.5cm;
+	padding-top: .5in;
+	padding-bottom: 2cm;
+	font-family: helvetica;} 
```

Comment:
Nice use of inheritance here to have your font family rule be used by the entire page.

Something to look at for next time - you can declare multiple values for font-family like this:
font-family: helvetica, arial, sans-serif;

Doing that basically means that if the browser cannot find the first font, it will try to use the next font. The "sans-serif" at the end will tell the browser to just use whatever default sans-serif font it can find if all else fails. (There is also a "serif" keyword you can use for the default serif font.)

Realistically, any computer should have helvetica, so this is a non-issue in this case.